ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity.component.ts

```diff
@@ -210,11 +213,13 @@ export class ActivityComponent extends BaseActivityComponent implements OnInit,
         private renderer: Renderer2,
         private submitService: SubmitAnnouncementService,
         private analytics: AnalyticsEventsService,
+        private session: SessionMementoService,
         @Inject(DOCUMENT) private document: any,
         // using Injector here as we get error using constructor injection
         // in both child and parent classes
         injector: Injector) {
         super(injector);
+        this.invitationId = this.session.session.invitationId;
```

Comment:
What I was referring to in comment above: subject-panel has its own reference to SessionMementoService, so don't think finding invitationId in this component helps us much.